doc/concepts/configuration/configuration_code.rst

```diff
@@ -0,0 +1,341 @@
+
+..  code-block:: console
+
+    export TT_REPLICATION="localhost:3301,localhost:3302"
+
+If you need to pass :ref:`additional parameters for URI <index-uri-several-params>`, use the ``?`` and ``&`` delimiters:
+
+..  code-block:: console
+
+    export TT_LISTEN="localhost:3301?param1=value1&param2=value2"
+
+An empty variable (``TT_LISTEN=``) has the same effect as an unset one meaning that the corresponding configuration parameter won't be set when calling ``box.cfg{}``.
+
+
+
+..  _index-local_hot_standby:
+..  _index-replication_port:
+..  _index-slab_alloc_arena:
+..  _index-replication_source:
+..  _index-snap_dir:
+..  _index-wal_dir:
+..  _index-wal_mode:
+..  _index-checkpoint daemon:
+
+..  _box_cfg_params:
+
+
+Configuration parameters
+------------------------
+
+Configuration parameters have the form:
+
+:extsamp:`{**{box.cfg}**}{[{*{key = value}*} [, {*{key = value ...}*}]]}`
+
+Since ``box.cfg`` may contain many configuration parameters and since some of the
+parameters (such as directory addresses) are semi-permanent, it's best to keep
+``box.cfg`` in a Lua file. Typically this Lua file is the initialization file
+which is specified on the Tarantool command line.
+
+Most configuration parameters are for allocating resources, opening ports, and
+specifying database behavior. All parameters are optional.
+A few parameters are dynamic, that is, they can be changed at runtime by calling ``box.cfg{}`` a second time.
+For example, the command below sets the :ref:`listen port <cfg_basic-listen>` to ``3301``.
+
+..  code-block:: tarantoolsession
+
+    tarantool> box.cfg{ listen = 3301 }
+    2023-05-10 13:28:54.667 [31326] main/103/interactive I> tx_binary: stopped
+    2023-05-10 13:28:54.667 [31326] main/103/interactive I> tx_binary: bound to [::]:3301
+    2023-05-10 13:28:54.667 [31326] main/103/interactive/box.load_cfg I> set 'listen' configuration option to 3301
+    ---
+    ...
+
+
+To see all the non-null parameters, execute ``box.cfg`` (no parentheses).
+
+..  code-block:: tarantoolsession
+
+    tarantool> box.cfg
+    ---
+    - replication_skip_conflict: false
+      wal_queue_max_size: 16777216
+      feedback_host: https://feedback.tarantool.io
+      memtx_dir: .
+      memtx_min_tuple_size: 16
+      -- other parameters --
+    ...
+
+To see a particular parameter value, call a corresponding ``box.cfg`` option.
+For example, ``box.cfg.listen`` shows the specified :ref:`listen address <cfg_basic-listen>`.
+
+..  code-block:: tarantoolsession
+
+    tarantool> box.cfg.listen
+    ---
+    - 3301
+    ...
+
+
+
+.. _index-uri:
+
+Listen URI
+----------
+
+Some configuration parameters and some functions depend on a URI (Universal Resource Identifier).
+The URI string format is similar to the
+`generic syntax for a URI schema <https://en.wikipedia.org/wiki/List_of_URI_schemes>`_.
+It may contain (in order):
+
+* user name for login
+* password
+* host name or host IP address
+* port number.
```

Comment:
(Looks unrelated to the changes in this PR, but I would highlight it.)

It seems, we should add query parameters into the list.